appcenter-crashes/scripts/postinstall.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+const path = require('path');
+
+const projectDirectory = path.resolve(`${__dirname}${path.sep}..${path.sep}..${path.sep}..`);
+const testDirectory = path.join(projectDirectory, 'test');
+const setupFileName = 'setupAppCenter.js';
+const packageJsonFile = path.join(`${projectDirectory}`, 'package.json');
+
+// Update project.json
+let packageJsonContent;
+try {
+    packageJsonContent = fs.readFileSync(packageJsonFile, 'utf8');
+} catch (e) {
+    console.log('Could not read package.json file');
+    return;
+}
+const projectJson = JSON.parse(packageJsonContent);
```

Comment:
nit - is there a reason to name this variable "projectJson" not "packageJson"?

Reply:
Just a typo, thanks!